<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>498</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="Slider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelPropeller">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 14pt &quot;MS Reference Sans Serif&quot;;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMax_eff">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>422</y>
     <width>60</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Eff</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  font: 10pt &quot;MS Reference Sans Serif&quot;;
  color: rgb(40, 50, 255);
}</string>
   </property>
   <property name="text">
    <string>Performance against J</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDfVar2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>adv ratio</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>x-axis</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QRadioButton" name="radioButtonRpm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>83</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>RPM</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonJ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>90</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Advance ratio</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelPitch">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>442</y>
     <width>60</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Pitch speed</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMaxeff">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>417</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*QLineEdit {
  border: none;
  border-bottom:1px solid;
  border-color:rgb(0, 170, 255);
  border-radius: 5;
  text-align: right;
 
}

#lineEditMaxeff:hover{
  border: 1.2px solid;
  border-color: rgb(180, 170, 200);
}

#lineEditMaxeff:focus{
  width:115;
  border: 1.2px solid;
  border-color: rgb(80, 170, 200);
}*/</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPitch">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>439</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelDfVar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>adv ratio</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>410</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>y-axis</string>
   </property>
   <widget class="QCheckBox" name="checkBoxCp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Cp</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {

}</string>
    </property>
    <property name="text">
     <string>Ct</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPower">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEditDfvar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>470</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="readyBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MV Boli&quot;;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="PlotWidget" name="widget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>Slider</zorder>
  <zorder>labelPropeller</zorder>
  <zorder>labelMax_eff</zorder>
  <zorder>label</zorder>
  <zorder>labelDfVar2</zorder>
  <zorder>groupBox</zorder>
  <zorder>labelPitch</zorder>
  <zorder>lineEditMaxeff</zorder>
  <zorder>lineEditPitch</zorder>
  <zorder>labelDfVar</zorder>
  <zorder>lineEditDfvar</zorder>
  <zorder>readyBox</zorder>
  <zorder>widget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
